my-uw/index.md

```diff
@@ -78,6 +78,219 @@ slug: my-uw
   <h4>App Description</h4>
   <p>80 characters maximum. Avoid the word “your.” (For example, instead of “view your earnings and tax statements,” use “view earnings and tax statements.”)</p>
   
+
+  <div class="page-header">
+    <h2>Widgets</h2>
+  </div>
+  <p>MyUW displays a grid of widgets on the home page in widget mode. You can easily create widgets using the <a href="https://github.com/UW-Madison-DoIT/myuw-smart-widget-creator">Widget Creator</a>. Each widget follows approximately this structure, but <i>widget developers only need to worry about the code inside the widget-body div.</i></p>
```

Comment:
"widget mode" -- should this be "smart widget mode", as differentiated from "minimal widget mode"?